src/components/about-us/flippingCard.jsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslations } from 'next-intl';
+import React from 'react';
+import styles from "@/styles/components/aboutus/flippingCard.module.scss";
+export async function getStaticProps({ locale }) {
+    return {
+        props: {
+            messages: (await import(`@/messages/${locale}.json`)).default
+        }
+    };
+}
+
```

Comment:
Since you'll be sending props in to the card, go ahead and move these lines into the corresponding `index.jsx` file